modules/phase_field/src/userobjects/DiscreteNucleationFromFile.C

```diff
@@ -34,8 +39,14 @@ DiscreteNucleationFromFile::DiscreteNucleationFromFile(const InputParameters & p
     _reader(getParam<FileName>("file")),
     _history_pointer(0),
     _tol(getParam<Real>("tolerance")),
-    _nucleation_rate(0.0)
+    _nucleation_rate(0.0),
+    _fixed_radius(getParam<bool>("fixed_radius")),
```

Comment:
```suggestion
    _fixed_radius(isParamValid("fixed_radius")),
```

Reply:
Wait, I thought "fixed_radius" was removed.  It should be "radius", right?